modules/ROOT/pages/cassandra/cassandra-xml-maven.adoc

```diff
@@ -0,0 +1,56 @@
+= XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Add a Namespace for the Connector
+<No Namespace>
+
+== Configure the Maven POM File
+When creating an app manually from the XML, you need to set up the pom.xml for your project:
+
+ [source,xml,linenums]
+----
+<repositories>
+   <repository>
+        <id>mule-ee-releases</id>
+        <name>MuleEE Releases Repository</name>
+        <url>https://repository-master.mulesoft.org/nexus/content/repositories/releases-ee/</url>
+    <repository>
+        <id>mule-ee-snapshots</id>
+        <name>MuleEE Snapshots Repository</name>
+        <url>https://repository-master.mulesoft.org/nexus/content/repositories/ci-snapshots/</url>
+    </repository>
+</repositories>
+----
+
+ This snippet specifies a project dependency for a specific release version:
+
+ [source,xml,linenums]
+----
+<dependency>
+    <groupId>org.mule.connectors</groupId>
+    <artifactId>mule-cassandra-connector</artifactId>
+    <version>RELEASE</version>
+    <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+ This snippet adds specifies a project dependency for the latest version:
+
+ [source,xml,linenums]
+----
+<dependency>
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-cassandra-connector</artifactId>
+    <version>LATEST</version>
+</dependency>
+----
+
+
+----
+<No Dependency Snippets>
```

Comment:
remove this source block since you list the pom file info already

Reply:
Fixed.